### PR TITLE
OCPBUGS#16887: Clarify cert-manager upgrade paths for users

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-16887

Link to docs preview: https://64946--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-install#cert-manager-operator-update-channels_cert-manager-operator-install

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:

### CODE REVIEW

**Review by lunarwhite — COMMENTED** [review]
/lgtm

cc @swghosh @thejasn @xingxingxia

File: modules/cert-manager-operator-update-channels.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_content-type: CONCEPT
+[id="cert-manager-operator-update-channels_{context}"]
+= Understanding update channels of the {cert-manager-operator}
```
Comment (dfitzmau) [inline modules/cert-manager-operator-update-channels.adoc:7]: Attributes render in module titles is usually fine but have any other modules used this format and proved to now impact how titles render on the customer portal?

File: modules/cert-manager-operator-update-channels.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* `stable-v1.y`
+
+[id="stable-v1-channel_{context}"]
+== stable-v1 channel
+
+The `stable-v1` channel is the default and recommended channel while installing the {cert-manager-operator}. The `stable-v1` channel installs and updates the latest release version of the {cert-manager-operator}. Select the `stable-v1` channel if you want to use the latest stable release of the {cert-manager-operator}.
+
+The `stable-v1` channel offers the following update approval strategies:
+
+Automatic:: If you choose automatic updates for an installed {cert-manager-operator}, when a new version of the {cert-manager-operator} is available in the `stable-v1` channel, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention.
```
Comment (dfitzmau) [inline modules/cert-manager-operator-update-channels.adoc:21]: This sentence has over 32 words. Is there a way to turn it into two sentences?

Screenshot from 2023-09-28 14-06-07

**Review by dfitzmau — COMMENTED** [review]
Nice work, @xenolinux . Well written content! Some minor comments, but the customer portal would be worth confirming. 

/remove-label peer-review-in-progress

/label peer-review-done

File: modules/cert-manager-operator-update-channels.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/cert-manager-operator-update-channels.adoc:7]: Yes this is the published change https://docs.openshift.com/container-platform/4.13/security/cert_manager_operator/cert-manager-operator-install.html

File: modules/cert-manager-operator-update-channels.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Update channels are the mechanism by which you can declare the version of your {cert-manager-operator} in your cluster. The {cert-manager-operator} offers the following update channels:
+
+* `stable-v1`
+* `stable-v1.y`
+
+[id="stable-v1-channel_{context}"]
+== stable-v1 channel
+
+The `stable-v1` channel is the default and recommended channel while installing the {cert-manager-operator}. The `stable-v1` channel installs and updates the latest release version of the {cert-manager-operator}. Select the `stable-v1` channel if you want to use the latest stable release of the {cert-manager-operator}.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/cert-manager-operator-update-channels.adoc]: Is "suggested" okay instead of "useful"?

[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/cert-manager-operator-update-channels.adoc]: Yes, @xenolinux , that works! Thanks.

**Pull request merged**